cunit/slowio.testc

```diff
@@ -112,6 +112,25 @@ static void test_limit2(void)
     }
 }
 
+void busywait_for(struct timespec *howlong)
```

Comment:
Can you make this function static, it doesn't need to be visible to other modules.  Like this:

```suggestion
static void busywait_for(struct timespec *howlong)
```